### PR TITLE
Refactor the Code For Netbeans UI designer 

### LINKED ISSUE #39: input field cropped in custom lookup dialog when label is long
When the label of an input field in the custom look dialog is too long, the input field gets pushed to the left and is cropped.

grafik

```xml 
<selection label="a-very-long-label-which-pushes-out-the-input-field">@n</selection>
```

### PR DESCRIPTION
**Pull request by marmoure**
While adding some features we had to do some manual changes which broke the Netbeans UI designer code, this PR should fix that And also Fixes #39 and now should allow for long labels.
image

### CODE REVIEW

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java [hunk 1]
```diff
@@ -96,16 +96,11 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         jLabel1.setText("Custom lookup");
 
         dependentJLabel.setText("Dependent:");
-        dependentJLabel.setToolTipText("");
-
-        dependentJTextField.addActionListener(new java.awt.event.ActionListener() {
```
Comment (adamretter) [inline src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java:101]: Did you mean to remove this listener?

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java [hunk 2]
```diff
@@ -84,7 +84,7 @@ private void initComponents() {
         restultsJTable = new javax.swing.JTable();
 
         setDefaultCloseOperation(javax.swing.WindowConstants.DISPOSE_ON_CLOSE);
-        setTitle("Custom lookup");
```
Comment (adamretter) [inline src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java:87]: Did you mean to remove the title?

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form [hunk 3]
```diff
@@ -100,35 +99,31 @@
...
+      <Properties>
+        <Property name="minimumSize" type="java.awt.Dimension" editor="org.netbeans.beaninfo.editors.DimensionEditor">
+          <Dimension value="[200, 22]"/>
+        </Property>
+      </Properties>
     </Component>
     <Component class="javax.swing.JButton" name="fetchjButton">
       <Properties>
-        <Property name="text" type="java.lang.String" value="Search"/>
+        <Property name="text" type="java.lang.String" value="Search..."/>
```
Comment (adamretter) [inline src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form:126]: Did you mean to rename the text on this button?

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form [hunk 4]
```diff
@@ -100,35 +99,31 @@
         <Property name="font" type="java.awt.Font" editor="org.netbeans.beaninfo.editors.FontEditor">
           <Font name="Segoe UI" size="18" style="0"/>
         </Property>
-        <Property name="text" type="java.lang.String" value="Custom Lookup"/>
+        <Property name="text" type="java.lang.String" value="Custom lookup"/>
```
Comment (adamretter) [inline src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form:102]: Did you mean to rename the text of this label?

**Review by adamretter — COMMENTED** [review]
Some of these changes perhaps aren't meant to be in this PR (or at least the title of the PR doesn't indicate that they should be)

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form (hunk 4 above)
Comment (marmoure) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form:102]: Yes

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form (hunk 3 above)
Comment (marmoure) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form:126]: Yes

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java (hunk 2 above)
Comment (marmoure) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java:87]: Yes

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java (hunk 1 above)
Comment (marmoure) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java:101]: Yes

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java (hunk 2 above)
Comment (adamretter) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java:87]: Erm... okay, but why?

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java (hunk 1 above)
Comment (adamretter) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java:101]: Okay... but why did you decide to remove it?

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form (hunk 3 above)
Comment (adamretter) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form:126]: Why is that? It doesn't seem related to the title of the PR?

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form (hunk 4 above)
Comment (adamretter) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.form:102]: Why is that? It doesn't seem related to the title of the PR?

**Comment by marmoure** [comment]
> Some of these changes perhaps aren't meant to be in this PR (or at least the title of the PR doesn't indicate that they should be)

Agree, I should have mentioned that while adding some features last time i broke the code for the visual designer, while trying to fix #39 i had to refactor the code a bit.

File: src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java (hunk 2 above)
Comment (marmoure) [reply src/main/java/org/humanistika/oxygen/tei/completer/GUI/newSuggestionForm.java:87]: This String is already Set in [newSuggestionForm.form](https://github.com/BCDH/TEI-Completer/pull/40/files#diff-7baf7dda0096af18fe7d4f2f0ec5622f3880cdb1dbca28ca87b92955c4680625) And this is now duplicated.

**Pull request merged**